x-pack/spec/modules/arcsight/yaml/smart_connector_with_ssl.yml

```diff
@@ -0,0 +1,10 @@
+modules:
+  - name: arcsight
+    var.inputs: "tcp"
```

Comment:
Would it make sense to use `smartconnector` rather than `tcp` to be consistent with filename and documentation?

Reply:
See above

Reply:
Adding comments to yaml